### ISSUE TITLE
[BUG] Cognitive.Vision.Face Identify MaxNumberOfCandidatesReturned doesn't match docs

### ISSUE THREAD

**Issue by danzel** [Azure/azure-sdk-for-net#14027]
**Describe the bug**
You cannot call Identify with maxNumberOfCandidatesReturned > 5

**Expected behavior**
The documentation says this value can be 1-100 with a default of 10.
https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395239
> maxNumOfCandidatesReturned (optional) | Number | The range of maxNumOfCandidatesReturned is between 1 and 100 (default is 10).

**Actual behavior (include Exception or Stack Trace)**
`Microsoft.Rest.ValidationException: 'maxNumOfCandidatesReturned' exceeds maximum value of '5'.`

**To Reproduce**
```
FaceClient.Face.IdentifyAsync(..., largePersonGroupId: largePersonGroupId, maxNumOfCandidatesReturned: 10);
```

**Environment:**
 - Microsoft.Azure.CognitiveServices.Vision.Face 2.5.0-preview.1
 - Azure AppService .net core
 - Visual Studio 16.4.6

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team best able to assist.

**Comment by KranthiPakala-MSFT** [comment]
@JinyuID , @dipidoo , @SteveMSFT - Could you please have a look at this feedback and share your thoughts? - Thank you

**Comment by danzel** [comment]
Looks like this was fixed in #25236
